tests/subtype/test_qc.py

```diff
@@ -2,18 +2,17 @@
 
 import pandas as pd
 
-from bio_hansel.qc import is_maybe_intermediate_subtype
-from bio_hansel.qc.const import QC
-from bio_hansel.subtype import Subtype
-from bio_hansel.subtyper import subtype_reads, subtype_contigs
-from bio_hansel.utils import init_subtyping_params
+from biohansel.subtype.qc import is_maybe_intermediate_subtype, QC
+from biohansel.subtype.subtype import Subtype
+from biohansel.subtype.subtyper import subtype_reads, subtype_contigs
+from biohansel.subtype.util import init_subtyping_params
 
 genome_name = 'test'
 
 
 def test_low_coverage():
     scheme = 'heidelberg'
-    fastq = 'tests/data/SRR1696752/SRR1696752.fastq'
+    fastq = 'tests/data/subtype/SRR1696752/SRR1696752.fastq'
     st, df = subtype_reads(reads=fastq, genome_name=genome_name, scheme=scheme)
```

Comment:
is "st" the standard for subtype for the file?

Reply:
Yeah, it should be fairly standard for a `Subtype` object to just be named `st` for brevity throughout the codebase.